src/otx/core/config/data.py

```diff
@@ -76,6 +76,7 @@ class TileConfig:
     max_num_instances: int = 1500
     object_tile_ratio: float = 0.03
     sampling_ratio: float = 1.0
+    with_full_img: bool = False
```

Comment:
Isn't the same comment applicable here?
https://github.com/openvinotoolkit/model_api/pull/174#discussion_r1609716047

Reply:
I set false as there's an OOM issue, that said, memory usage improvement will come later.